sample/build.gradle.kts

```diff
@@ -90,6 +91,10 @@ sourceSets {
 
 tasks.withType<KotlinCompile> {
     kotlinOptions {
-        jvmTarget = "1.8"
+        jvmTarget = "17"
     }
 }
+
+tasks.withType<SwaggerCodeGenTask> {
+    project.tasks.findByName("kaptGenerateStubsKotlin")?.dependsOn(this)
```

Comment:
Why do you create dependency on `kaptGenerateStubsKotlin`? I think it should depend on `KotlinCompile` so for example:
```kotlin
tasks.withType<KotlinCompile<*>> {
    dependsOn(/*name of tasks*/)
}
```

Reply:
This is to resolve following issue when trying to build the sample.

A problem was found with the configuration of task ':sample:swagger-kotlin-petstore-def-and-opt-query' (type 'SwaggerCodeGenTask').
  - Gradle detected a problem with the following location: '/Users/kuba/Documents/work/projects/swagger-fork/sample/build/openapi/petstore-def-and-opt-query'.
    
    Reason: Task ':sample:kaptGenerateStubsKotlin' uses this output of task ':sample:swagger-kotlin-petstore-def-and-opt-query' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed.
    
    Possible solutions:
      1. Declare task ':sample:swagger-kotlin-petstore-def-and-opt-query' as an input of ':sample:kaptGenerateStubsKotlin'.
      2. Declare an explicit dependency on ':sample:swagger-kotlin-petstore-def-and-opt-query' from ':sample:kaptGenerateStubsKotlin' using Task#dependsOn.
      3. Declare an explicit dependency on ':sample:swagger-kotlin-petstore-def-and-opt-query' from ':sample:kaptGenerateStubsKotlin' using Task#mustRunAfter.
    
    For more information, please refer to https://docs.gradle.org/8.4/userguide/validation_problems.html#implicit_dependency in the Gradle documentation.

Reply:
Okey 👍